props/saber_fett263_buttons.h

```diff
@@ -257,7 +325,100 @@ SaberFett263Buttons() : PropBase() {}
       case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_OFF | BUTTON_AUX):
         previous_preset();
         return true;
+
+       // Gesture Controls
+		
+       // Battle Mode
+		    
+       #ifdef FETT263_BATTLE_MODE
+       case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_ON | BUTTON_AUX):
+           if (!battle_mode_) {
+              battle_mode_ = true;
+              hybrid_font.SB_Force();
+           } else {
+              battle_mode_ = false;
+              hybrid_font.SB_Force();
+           }
+           return true;
+
+       // Auto Lockup Mode
+       case EVENTID(BUTTON_NONE, EVENT_CLASH, MODE_ON):
+           if (battle_mode_) {
+              clash_impact_millis_ = millis();
+              swing_blast_ = false;
+              if (clash_impact_millis_ > last_swing_) {
```

Comment:
I'm not sure I understand what this is supposed to do.
Maybe this is supposed to be if (!swinging_) { ??

Reply:
this is checking the swing actual stopped (avoid clashes from sound effects like accent swing and/or spins) the blade has to impact (stop) to actually trigger, when you hit something your swing is stopped,